Authenticator/Classes/OTPTokenListViewController.swift

```diff
@@ -206,16 +206,10 @@ extension OTPTokenListViewController /* UITableViewDelegate */ {
     }
 
     override func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
+        let keychainItem = self.tokenManager.keychainItemAtIndex(indexPath.row)
```

Comment:
Length Violation: Line should be 100 characters or less: currently 101 characters